kube/aks/scheduler-shell.yaml

```diff
@@ -0,0 +1,89 @@
+---
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 Collabora Limited
+# Author: Guillaume Tucker <guillaume.tucker@collabora.com>
```

Comment:
I guess you to update Copyright year and Author?

Reply:
Done, i will keep Guillaume Tucker too as i mostly copied his template.